1_fetch/src/1_MendotaPull.R

```diff
@@ -0,0 +1,20 @@
+## This script pulls down model evaluation data for Lake Mendota
+
+## Function for pulling down data from 
+## USGS's ScienceBase https://www.sciencebase.gov/
+
+fetch_data <- function(ID, project_output_dir, file_name){
+  require(sbtools)
```

Comment:
Let's keep `require()` calls out of the pipeline because they won't be sufficiently visible to **targets**. The course will show you how to formally declare package dependencies soon.